server/src/main/java/jass/server/entity/GameEntity.java

```diff
@@ -0,0 +1,55 @@
+package jass.server.entity;
+
+import com.j256.ormlite.field.DatabaseField;
+import com.j256.ormlite.table.DatabaseTable;
+import jass.lib.database.Entity;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * A model with all known (and cached) Games.
+ *
+ * @author Thomas Weber
+ * @version %I%, %G%
+ * @since 0.0.1
+ */
+
+@DatabaseTable(tableName = "game")
+public class GameEntity implements Entity {
+
+    private static final Logger logger = LogManager.getLogger(GameEntity.class);
+
+    @DatabaseField(generatedId = true)
```

Comment:
I realised that we probably need to delete games immediately from the db when they're no longer being played. Otherwise we'll lose track of what games are active. 
Altrnatively we can add a boolean flag here "isActive"